starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2102,7 +2102,7 @@ sub print_report {
 	else{
 	    $o .= sprintf("<TD><img align=\"top\" src=\"%s/down.gif\" ></TD>", 
 			  $self->{STARCHECK} );
-	    $o .= sprintf("<TD>NEXT </TD>", $self->{next}->{obsid});
+	    $o .= sprintf("<TD>NEXT </TD>");
```

Comment:
I suppose I should just cut the NEXT completely if there isn't actually a next to go to...